sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Diagnostics/AzureMonitorExporterEventSource.cs

```diff
@@ -268,7 +272,15 @@ public void TransmitterFailed(string instrumentationKey, Exception ex)
         public void TransmitterFailed(string instrumentationKey, string exceptionMessage) => WriteEvent(33, instrumentationKey, exceptionMessage);
 
         [NonEvent]
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        internal bool IsEnabled(EventLevel eventLevel) => IsEnabled(eventLevel, EventKeywords.All);
+        public void BackoffEnabled(TimeSpan backoffInterval)
```

Comment:
Why is this needed?

Reply:
To help investigate "no data" scenarios.
The Exporter will backoff for a period of up to 1 hour.